RamilyaNigmatullina/pdp-3
# Add ability to subscribe to daily newsletters

# Summary

Add ability to subscribe to daily newsletters

https://github.com/RamilyaNigmatullina/pdp-3/projects/1#card-19568255

# Test plan

List of steps to manually test introduced functionality:

* Go to Application
* Fill in your email in the right bottom corner
* Click on subscribe
* Check your mailbox every day

# Review notes

While reviewing pull-request (especially when it's your pull-request),
please make sure that:

- you understand what problem is solved by PR and how is it solved
- new tests are in place, no redundant tests
- DB schema changes reflect new migrations
- newly introduces DB fields have indexes and constraints
- routes are RESTful, no useless routes
- there are no missed files (migrations, view templates)
- required ENV variables added and described in `.env.example` and added to Heroku
- associated Heroku review app works correctly with introduced changes

# Deploy notes

Add `MAILING_TIME`, `SALT` variables to the ENV.
